notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>Is the taxi data only available from 2016? (if so, how annoying!) if not, can we do it for all of the years after they did the taxi zones?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>The Taxi dataset has data in between 2015 and 2018 and each dataset has 100M rows, while the Citi Bike dataset has 53M rows in between 2013 and 2018. The major reason that I'm using 2016's data only is because it's too large. This project already filtered out most of data and use only about 10% of them since it consumes too much time and computing resource, we use only one years instead of all of the years.</p>